src/NLog/Targets/Wrappers/DebugTargetWrapper.cs

```diff
@@ -0,0 +1,76 @@
+﻿// 
+// Copyright (c) 2004-2016 Jaroslaw Kowalski <jaak@jkowalski.net>, Kim Christensen, Julian Verdurmen
+// 
+// All rights reserved.
+// 
+// Redistribution and use in source and binary forms, with or without 
+// modification, are permitted provided that the following conditions 
+// are met:
+// 
+// * Redistributions of source code must retain the above copyright notice, 
+//   this list of conditions and the following disclaimer. 
+// 
+// * Redistributions in binary form must reproduce the above copyright notice,
+//   this list of conditions and the following disclaimer in the documentation
+//   and/or other materials provided with the distribution. 
+// 
+// * Neither the name of Jaroslaw Kowalski nor the names of its 
+//   contributors may be used to endorse or promote products derived from this
+//   software without specific prior written permission. 
+// 
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+// AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE 
+// IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE 
+// ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE 
+// LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR 
+// CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+// SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS 
+// INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN 
+// CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) 
+// ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF 
+// THE POSSIBILITY OF SUCH DAMAGE.
+// 
+
+namespace NLog.Targets.Wrappers
+{
+    using System;
+    using System.ComponentModel;
+    using System.Threading;
+    using NLog.Common;
+    using NLog.Internal;
+
+    /// <summary>
+    /// Wrapper for unit testing with DebugTarget
+    /// </summary>
+    [Target("DebugWrapper", IsWrapper = true)]
+    public class DebugTargetWrapper : WrapperTargetBase
```

Comment:
Please add this one to the test project, thx!

Reply:
Are you asking for that class to be moved into the testing namespace or for unit tests to be created for the class? I see the DebugTarget exists in the main project. Thanks in advance for the clarification.

Reply:
Move to the test project please, so it's only for the unit tests visible. 

You're right about the debug target. But I prefer to only add public surface with a feature request.

Reply:
The target being in the test project does not seem to work. I get a target not found. The same thing originally was occurring (and still is) when i tried to use the MockTargetWrapper. It appears that only targets in the main project are being found.

Reply:
Finished. Adding the assembly reference to unit test project in the nlog config string was enough.